polygeist/tools/cgeist/Lib/ValueCategory.h

```diff
@@ -46,6 +52,16 @@ class ValueCategory {
                         mlir::Type PromotionType) const;
 
   ValueCategory FPExt(mlir::OpBuilder &builder, mlir::Type PromotionType) const;
+  ValueCategory IntCast(mlir::OpBuilder &builder, mlir::Type PromotionType,
```

Comment:
[nit]: The .cc file names the parameter as `OpBuilder &Builder, Type PromotionType`, suggests to use capital letters in the header file as well.